apps/store/extensions/app/social-reviews/themes/store/css/theme-wso2.css

```diff
@@ -0,0 +1,168 @@
+@charset "UTF-8";
+/*
+~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/9a57d06aacf29863ac963c264ca95a8fba0efb00)